src/persistentmodel/job.h

```diff
@@ -69,18 +83,26 @@ class Job : public QObject, public PersistentObject
 
     bool settingHideSymlinks() const;
     void setSettingHideSymlinks(bool settingHideSymlinks);
+    //! @}
 
+    //! Begins performing a backup of this Job.
```

Comment:
This only creates a backupTask() with the properties of this Job, the backup is not initiated yet.